car_service/request_service.rb

```diff
@@ -0,0 +1,118 @@
+require_relative 'request_repo'
+
+#computed in seconds
+module TimeUtils
```

Comment:
I know this is debatable, but the ruby style guide encourages developers to use one class/module per file => https://github.com/rubocop-hq/ruby-style-guide#one-class-per-file